invenio_jsonschemas/ext.py

```diff
@@ -165,6 +167,17 @@ def path_to_url(self, path):
         ).build(
             'schema', values={'path': path}, force_external=True)
 
+    @cached_property
+    def loader(self):
+        """Loader used by `JsonRef.replace_refs`."""
+        try:
+            loader_ = import_string(self.app.config['JSONSCHEMAS_LOADER_CLS'])
+        except ImportError:
+            loader_ = json_loader_factory(jsonresolver.JSONResolver(
```

Comment:
Could we live with default loader that uses direct downloads and internal cache?